addons/sale_timesheet/models/project_sale_line_employee_map.py

```diff
@@ -34,6 +35,11 @@ class ProjectProductEmployeeMap(models.Model):
         ('uniqueness_employee', 'UNIQUE(project_id,employee_id)', 'An employee cannot be selected more than once in the mapping. Please remove duplicate(s) and try again.'),
     ]
 
+    @api.depends('employee_id', 'project_id.sale_line_employee_ids.employee_id')
+    def _compute_existing_employee_ids(self):
+        for map_entry in self:
+            map_entry.existing_employee_ids = map_entry.project_id.sale_line_employee_ids.employee_id
```

Comment:
we could do it in batch since we could be sure in most of the case we are in only one project.
```suggestion
        project = self.project_id
        if len(project) == 1:
            self.existing_employee_ids = project.sale_line_employee_ids.employee_id
            return
        for map_entry in self:
            map_entry.existing_employee_ids = map_entry.project_id.sale_line_employee_ids.employee_id
```